modules/nw-multus-remove-pod.adoc

```diff
@@ -0,0 +1,65 @@
+// Module included in the following assemblies:
+//
+
+[id="nw-multus-remove-pod_{context}"]
+= Removing a Pod from an additional network
+
+You can remove a Pod from an additional network.
+
+.Prerequisites
+
+* Install the OpenShift Command-line Interface (CLI), commonly known as `oc`.
+* You must log in to the cluster.
+
+.Procedure
+
+To add a Pod to an additional network, complete the following steps:
+
+. Edit the Pod resource definition by running the following command. Replace
+`<name>` with the name of the Pod to edit.
++
+----
+$ oc edit pod <name>
+----
+
+. Update the `annotations` mapping to remove the additional network from the
+Pod:
+
+.. If you are removing all additional networks from a Pod, remove the
```

Comment:
These should be in a ul instead of an ol - you do one, not both. Make this clearer in the parent step.

Reply:
s/If you are removing/To remove